src/switch_core_media.c

```diff
@@ -10130,14 +10145,27 @@ SWITCH_DECLARE(switch_status_t) switch_core_media_activate_rtp(switch_core_sessi
 
 		if (switch_channel_test_flag(session->channel, CF_TEXT_POSSIBLE) && t_engine->cur_payload_map->rm_encoding && t_engine->cur_payload_map->remote_sdp_port) {
 			/******************************************************************************************/
+
+			// Use static global default from RTP poll timeout
+			uint32_t poll_timeout_s = TELNYX_RTP_DEFAULT_POLL_TIMEOUT_S;
```

Comment:
This is redundant code, we can centralized the initialization for poll_timeout_s for audio, video and text